spec/unit/database_spec.rb

```diff
@@ -10,38 +10,15 @@
   end
 
   describe "#connect" do
```

Comment:
You are no longer describing `#connection`, so maybe change that...

`describe "@connection" do ... end ` ?